app/src/org/commcare/activities/MenuBase.java

```diff
@@ -74,7 +83,50 @@ protected boolean onBackwardSwipe() {
     }
 
     @Override
+    public boolean onCreateOptionsMenu(android.view.Menu menu) {
+        super.onCreateOptionsMenu(menu);
+        addHomeScreenActionsToTopBar(menu);
+        return true;
+    }
+
+    private void addHomeScreenActionsToTopBar(android.view.Menu menu) {
+        if (menuIsBeingUsedAsHomeScreen()) {
+            ViewUtil.addItemToActionBar(menu, MENU_LOGOUT, MENU_GROUP_HOME_SCREEN_ACTIONS, "Logout",
+                    R.drawable.ic_logout_action_bar);
+            ViewUtil.addItemToActionBar(menu, MENU_SYNC, MENU_GROUP_HOME_SCREEN_ACTIONS, "Sync",
+                    R.drawable.ic_sync_action_bar);
```

Comment:
Sorry that sent prematurely.. No, the consumer apps flag is separate, and for that we still don't want any notion of any of the home screen actions.